frontend/lib/api/brain/brain.ts

```diff
@@ -26,7 +25,7 @@ export const createBrain = async (
 };
 
 export const getBrain = async (
-  brainId: UUID,
```

Comment:
Why ?

Reply:
- Had to type all IDs as UUID or convert them to strings to avoid type mismatch errors in TypeScript.
- Chose the easier option of converting them to strings (`UUID => str`) since strings were more commonly used.
- The UUIDs didn't seem to provide much value in the current context.
- Open to the proposition of moving all IDs to UUID format for improved consistency.